package proto_ff;

import "yd_fieldoptions.proto";


message titletype
{
	optional int32 type = 1[(yd_fieldoptions.field_cname) = "称号类型"];
}


message Sheet_titletype
{
	repeated titletype titletype_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message titletitleattributeDesc
{
	optional int32 type = 1[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message titletitle
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "称号ID"];
	optional string name = 2[(yd_fieldoptions.field_cname) = "称号名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 type = 3[(yd_fieldoptions.field_cname) = "称号类型"];
	optional int32 subType = 4[(yd_fieldoptions.field_cname) = "获取条件类型"];
	optional string eventValue = 5[(yd_fieldoptions.field_cname) = "获取条件值", (yd_fieldoptions.field_bufsize) = 60];
	optional string describe = 6[(yd_fieldoptions.field_cname) = "获取条件描述", (yd_fieldoptions.field_bufsize) = 120];
	optional int32 addTime = 7[(yd_fieldoptions.field_cname) = "自动延期"];
	optional int32 lifeTime = 8[(yd_fieldoptions.field_cname) = "生效时间"];
	repeated titletitleattributeDesc attribute = 9[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 4];
}


message Sheet_titletitle
{
	repeated titletitle titletitle_List = 1[(yd_fieldoptions.field_arysize)=120];
}
